### ISSUE TITLE
Fix `license_tool` warning during `make distcheck`

### ISSUE THREAD

**Issue by dmcdougall** [libqueso/queso#409]
Here's the warning:

```
[license_tool]: Warning -> unable to create tmp file locally (/scratch/damon/git/github/queso/queso-0.54.0/_build/../examples/validationCycle2/src/.tga2_func.C.tmp) - aborting update.
```

The solution (thanks @pbauman) is to not include our license generation infrastructure in the distribution.  It's not clear what side effects this will have.

**Comment by pbauman** [comment]
I've done this in both [Antioch](https://github.com/libantioch/antioch/pull/197/commits/04df34e059dba385c1373440747810d4f1a3455e) and [GRINS](https://github.com/grinsfem/grins/pull/408/commits/9329336932f7741d53dbc2c9b5fb51b17cbc351c) with no problems.

**Comment by dmcdougall** [comment]
Fixed in #470.
